components/automate-ui/src/app/pages/service-groups/service-groups.component.ts

```diff
@@ -294,11 +294,20 @@ export class ServiceGroupsComponent implements OnInit, OnDestroy {
   detailParamsChange(queryParams) {
     const sgId = queryParams.get('sgId');
     if (sgId) {
+      const searchBarFilters = [];
+      const paramKeys = Object.keys(queryParams.params);
+      paramKeys.forEach((key: string) => {
+        if (some({'type': key}, this.categoryTypes)) {
+          searchBarFilters.push({ type: key, text: queryParams.params[key] });
```

Comment:
I'd prefer 
```typescript
const searchBarFilters = paramKeys.filter((key: string) => some({'type': key}, this.categoryTypes))
  .map((key: string) => ({ type: key, text: queryParams.params[key] }));
```
but it's just a matter of taste 😃

Reply:
@srenatus I like it 👍🏻